drivers/sensor/lis2dw12/lis2dw12.c

```diff
@@ -238,6 +239,41 @@ static int lis2dw12_attr_set_thresh(const struct device *dev,
 }
 #endif
 
+#if CONFIG_LIS2DW12_FREEFALL
```

Comment:
#ifdef

Reply:
i think also it should be #ifdef. but #if is used for other preprocessor commands in this file. So i used if instead of ifdef.

Reply:
Resolved.